docker-compose.yml

```diff
@@ -54,55 +62,66 @@ services:
       CHARON_DATA_DIR: /charon/bootnode
       CHARON_P2P_BOOTNODES: ""
       CHARON_P2P_EXTERNAL_HOSTNAME: bootnode
-
-  vc0-lh:
+    volumes:
+      - ./:/charon/
+  #             _ _     _       _                 
+  # __   ____ _| (_) __| | __ _| |_ ___  _ __ ___ 
+  # \ \ / / _` | | |/ _` |/ _` | __/ _ \| '__/ __|
+  #  \ V / (_| | | | (_| | (_| | || (_) | |  \__ \
+  #   \_/ \__,_|_|_|\__,_|\__,_|\__\___/|_|  |___/
+  vc0:
     build: lighthouse
     networks: [cluster]
     depends_on: [node0]
+    env_file: .env
     environment:
       NODE: node0
     volumes:
-      - .:/charon
+      - ./lighthouse:/opt/charon/lighthouse
+      - .charon/cluster/node0:/opt/charon/keys
 
-  vc1-teku:
-    image: consensys/teku:latest
+  vc1:
+    image: consensys/teku:22.5.0
     networks: [cluster]
     depends_on: [node1]
+    restart: on-failure
     command: |
       validator-client
-      --network=auto
-      --beacon-node-api-endpoint="http://node1:16002" 
-      --validator-keys="/charon/node1:/charon/node1"
-      --validators-proposer-default-fee-recipient="0x0000000000000000000000000000000000000000"
+      --beacon-node-api-endpoint="http://node1:16002"
+      --config-file "/opt/charon/teku/teku-config.yaml"
     volumes:
-      - .:/charon
+      - .charon/cluster/node1:/opt/charon/keys
+      - ./teku/:/opt/charon/teku
 
-  vc2-teku:
```

Comment:
i think we should keep this as it helps to checkout in docker-compose logs what exactly are the logs from teku/lighthouse

Reply:
Hmm yeah I can try. I removed it while trying to debug why I couldn't hit the teku metrics endpoint.